spec/requests/api/docs/openapi_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe API::Docs::OpenapiController, type: :request do
+  describe 'GET#show' do
+    context 'given an request' do
```

Comment:
```suggestion
    context 'given a request' do
```

Reply:
Fixed in https://github.com/nimblehq/nimble-survey-web/pull/89/commits/43a2d65fc23d5253012f31f0b2fcdc505e1b3511